src/triage/component/architect/builders.py

```diff
@@ -305,10 +310,36 @@ def build_matrix(
                 format='csv'
             )
             logging.info("Matrix {matrix_uuid} archived (using metta)")
+            # If completely archived, save its information to matrices table
+            # At this point, existence of matrix already tested, so no need to delete from db
+            if matrix_type == 'train':
+                lookback = matrix_metadata["max_training_history"]
+            else:
+                lookback = matrix_metadata["test_duration"]
+
+            matrix = Matrices(
+                matrix_id = matrix_metadata["matrix_id"],
```

Comment:
No spaces around the = here too

https://stackoverflow.com/questions/8853063/pep-8-why-no-spaces-around-in-keyword-argument-or-a-default-parameter-value